farmers-hub/src/app/(app)/api/stripe/webhooks/route.ts

```
@@ -84,6 +91,19 @@ export async function POST(req: Request) {
+          data = event.data.object as Stripe.Account;
+          await payload.update({
+            collection: "tenants",
+            where: {
+              stripeAccountId: {
+                equals: data.id,
+              },
+            },
+            data: {
+              stripeDetailsSubmitted: data.details_submitted,
+            },
+          });
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)